zombienet-sdk-tests/tps/.rustfmt.toml

```diff
@@ -0,0 +1,24 @@
+# Basic
```

Comment:
The project already contains a `.rustfmt.toml` file in its root folder. Shouldn't that one be reused here?

Reply:
Yes, this crate is not part of the workspace but we can add a symlink here.